narayana-bom/pom.xml

```diff
@@ -57,11 +57,11 @@
     <version.org.jboss.invocation>2.0.0.Final</version.org.jboss.invocation>
     <version.org.jboss.jandex>2.1.1.Final</version.org.jboss.jandex>
     <version.org.jboss.jboss-transaction-spi>8.0.0.Final</version.org.jboss.jboss-transaction-spi>
-    <version.org.jboss.logging.jboss-logging>3.5.0.Final</version.org.jboss.logging.jboss-logging>
-    <version.org.jboss.logging.jboss-logging-processor>2.2.1.Final</version.org.jboss.logging.jboss-logging-processor>
-    <version.org.jboss.logmanager>1.5.1.Final</version.org.jboss.logmanager>
-    <version.org.jboss.openjdk-orb>9.0.1.Final</version.org.jboss.openjdk-orb>
-    <version.org.jboss.ws>2.0.0.Final</version.org.jboss.ws>
+    <version.org.jboss.logging.jboss-logging>3.6.1.Final</version.org.jboss.logging.jboss-logging>
+    <version.org.jboss.logging.jboss-logging-processor>3.0.2.Final</version.org.jboss.logging.jboss-logging-processor>
+    <version.org.jboss.logmanager>3.0.6.Final</version.org.jboss.logmanager>
```

Comment:
I think it's possible that WildFly has not got to this version yet: https://github.com/wildfly/wildfly-core/blob/27.0.0.Beta1/pom.xml#L218

Reply:
You are right, I'll update it to 2.1.19.Final then. Thanks